Slight Screen modification and ftFT command changes

RiordanIX opened the pull request:
'f','t','F',and 'T' were all still using cin.get() to get the second character, which was causing Aborts and other bad behavior.  It's been changed to use the new way to get character inputs.

Also, the Cursor was moving all over the place whenever a movement key was held down, so it now hides the cursor until movement is over.

RiordanIX committed:
Changed input for t,T,f,F

t,T,f and F were still using cin.get() and it caused very strange
things.  It now uses the same function to read a key as almost
everything else.

RiordanIX committed:
Cursor no longer shoots around all over the place

The Cursor was moving around all over the place when holding down
scrolling buttons.  It's now hidden whenever it's moving so you don't
see it moving in weird places, but reappears when you aren't scrolling
anymore.

tomsmeding commented on io.cpp:
```diff
@@ -861,12 +861,14 @@ bool jumpToNextOccurrenceOfChar(Inter::Filebuffer &fbuf,char c,unsigned int skip
 
 bool jumpToPreviousOccurrenceOfChar(Inter::Filebuffer &fbuf,char c,unsigned int skip=0){
 	unsigned int i;
+	cerr <<"This does stuff";
```
And now it also prints things to standard error ;)

RiordanIX replied:
Heh, yeah.  I forgot I put those there.  I forget even why now.

tomsmeding commented:
Hi, thanks for your fixes. I made some remarks here and there, could you have a look?

I'll be quite busy the next few weeks, so don't be surprised if I don't respond for some time :)

RiordanIX committed:
Added HOME and END movements.  Removed cerr

Minor change to moving back from F and T.  HOME and END now work
in insert and normal modes. Removed cerr and modified hiding the
cursor with gettput() instead of printf().

RiordanIX commented:
I saw the remarks and fixed cerr and changed printf() with gettput().  I also noticed that HOME and END weren't implemented so i implemented them.

tomsmeding commented:
Looks cool!
